pkg/config/config.go

```diff
@@ -75,9 +75,17 @@ type EventstoreConfig struct {
 	MySQL             *MySQLConfig    `mapstructure:"mysql"`
 	Postgres          *PostgresConfig `mapstructure:"postgres"`
 	SQLite            *SQLiteConfig   `mapstructure:"sqlite"`
+	Tigris            TigrisConfig    `mapstructure:"tigris"`
 	SynchronizeEvents bool            `mapstructure:"synchronizeEvents"`
 }
 
+type TigrisConfig struct {
+	URL          string `mapstructure:"url"`
+	Project      string `mapstructure:"project"`
+	ClientID     string `mapstructure:"client_id"`
```

Comment:
Since our project uses camelCase for property names, can we update the `mapstructure` tag for `ClientID` to be `mapstructure:"clientId"` and for `ClientSecret` to be `mapstructure:"clientSecret"`?